mgl-webapp/src/main/java/com/organization/mvcproject/controller/MGLController.java

```diff
@@ -34,48 +33,45 @@ public String home() {
 	@RequestMapping(value = "/review", method = RequestMethod.GET)
 	public ModelAndView review() {
 	/**
-	 * TODO 1.0 Rename the jsp view, to "reviewCreatePage" because it matches the URL triggering a circular view path error.
+	 * 
 	 * update games.jsp as well. 
 	 * SEE:  https://www.baeldung.com/spring-circular-view-path-error
 	 */
-		return new ModelAndView("review", "command", new Review());
+		return new ModelAndView("createReview", "command", new Review());
 	}
 
 	@RequestMapping(value = "/addReview", method = RequestMethod.POST)
 	public ModelAndView addReview(Review review, ModelMap model) {
 		if(review.getAuthor().equals("")) {
 			review.setAuthor("anonymous");
 		}
-	/**
-	 * TODO 1.0 Rename the jsp view, to "reviewDetailPage" because what is the view the "result" of?
-	 * update games.jsp as well. 
-	 */
-		return new ModelAndView("result", "submittedReview", review);
+
+		return new ModelAndView("reviewResult", "submittedReview", review);
 	}
 
 	
 	@RequestMapping(value = "/games", method = RequestMethod.GET)
 	public ModelAndView game() {
 		/**
-		 * TODO 1.0 Rename the jsp view, to "gamesPage" because it matches the URL triggering a circular view path error.
+		 * 
 		 * update games.jsp as well. 
 		 * SEE:  https://www.baeldung.com/spring-circular-view-path-error
 		 */
-		return new ModelAndView("games", "command", new Game());
+		return new ModelAndView("gamesPage", "command", new Game());
 	}
 
 	/**
 	 * TODO 2.0 (Separation of concerns) consider moving all controller endpoints that return a ResponseEntity into a @RestController.
 	 */
 	
 	//TODO 1.0 RequestMapping URL should follow RESTful.
-	@RequestMapping(value = "/game/getAll", method = RequestMethod.GET)
+	@RequestMapping(value = "/game/get", method = RequestMethod.GET)
```

Comment:
do you think you could find a way to not have to repeat /game for each requestMapping?